plots/laws/quantum_mechanics/harmonic_oscillatior_eigenfunctions.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+from sympy import pi
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import units, convert_to
+from symplyphysics.laws.quantum_mechanics.harmonic_oscillator import wave_eigenfunctions
+
+values = {
+    wave_eigenfunctions.oscillator_mass: convert_to(units.planck_mass, units.kilogram).evalf(),
+    wave_eigenfunctions.angular_frequency: 1e10,  # rad/s
+}
+
+mode_numbers_ = 0, 1, 2, 3
+
+base_plot = plot(
+    title="Quantum harmonic oscillator eigenfunctions",
+    xlabel=r"position, $\text{m}$",
+    ylabel=r"wave function, $\text{m}^{-1/2}$",
+    backend=MatplotlibBackend,
+    legend=True,
+    show=False,
+    annotations=None,
+)
+
+law = wave_eigenfunctions.law.rhs.subs({
+    pi: pi.evalf(),
+    wave_eigenfunctions.hbar: convert_to(units.hbar, units.joule * units.second).evalf(),
+}).evalf()
```

Comment:
Why this `evalf`?

Reply:
At first there were some problems with lambdifying expressions so I `evalf`ed everything. But looks like it's fine without it too